docs/Documentation.csproj

```diff
@@ -1,7 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project>
 
+  <!-- See https://github.com/microsoft/MSBuildSdks/tree/master/src/NoTargets -->
+  <Sdk Name="Microsoft.Build.NoTargets" Version="1.0.73" />
+  
   <ItemGroup>
     <None Include="**/*.md" />
+    <PackageReference Remove="@(PackageReference)" />
```

Comment:
This removes our default references (via `Directory.Build.props`) to make the Dependencies node nicer here. It's not required.